### ISSUE TITLE
[a11y Design Review] MHV: Cartography Team: Account Creation API Error Alerts - Staging

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#99092]
> [!NOTE]
> **QUICK SUMMARY:**
>
> - **Team/Item:** MHV Cartography: Account Creation API Error Alerts
> - **Timeline:** As soon as possible
> - **Dependencies**: `Yes`- the shipment relies not only on Cartography's implementation of these alerts, but in the implementation on 3 other affected tool applications as well (Medications, medical records, and secure messages).
> - **Review 1**: Staging evaluation of these alerts in-place on the My HealtheVet landing page
> - **Review 2**: Also an assessment of whether there would be any issues with placement* / focus management order of these alerts on the main entry point pages of affected tool applications (Medications, medical records, and Secure Messages). *_The placement on these pages will be different than the My HealtheVet landing page._

### Background

> MHV Cartography needs a thorough accessibility review of **Account Creation API Error Alerts.**

An overview of this project is available [here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/account-creation-api.md) but in a nutshell: there is a problem for users who create new sign-in credentials who haven't used the Classic version of My HealtheVet previously, that they won't have an identifier with a specific API (MHV-API) that data comes from for 3 of the "big 4" health tools: Medications, Medical records, and Secure Messages.

To get around this, and the need to ask users to manually register for an Identifier, a new API has been built that will fire-off when a user signs in to VA.gov and create an Identifier for them in the background. This should solve the vast majority of issues with users not having access to those 3 tools when they should. However, it is possible that the API could throw an error.

If the API does throw an error, there are two "groups" of errors. Group 1.) User-action required errors can only be resolved if the user calls the My HealtheVet help desk; and Group 2.) background errors that are just random API problems that probably won't happen again and won't be solved by a user calling a help desk.

We have designed 2 alerts to cover these two groups. They are available in Figma (linked below under supporting artifacts) & are being built out by the Cartography team right now.

### Collaboration timeframe 

> Our goal is to ship these alerts as soon as possible, since they will replace a currently terrible user experience for Veterans. 

Our goal is to ship these alerts as soon as possible, since they will replace a currently terrible user experience for Veterans. Happy to discuss more in person! But the shipment relies not only on Cartography's implementation of these alerts, but in the implementation on 3 other affected tool applications as well (Medications, medical records, and secure messages).

## About the Team

- Product name: My HealtheVet on VA.gov
- Team name: Cartography Team
- OCTO product owner: Robyn Singleton
- Product manager: Wes Rowe
- IA: Sara Sterkenburg
- Slack channel: #mhv-on-vagov-cartography-team

### Supporting Artifacts 
- [x] [Product brief / overview of the Account Creation API work](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/account-creation-api.md)
- [x] [Figma files](https://www.figma.com/design/m992k2m1DSl9MXV9hDytsQ/MHV-Account-Security-%26-Sign-In?node-id=229-3&node-type=canvas&t=0f88dTRaftma6YpE-0)
- [ ] Staging Link `Needed`

```[tasklist]
### a11y Tasks
- [x] Tag @artsymartha68 in ticket for priority level assessment
- [x] Receive priority level assessment from @artsymartha68
- [ ] Collaborate with MHV Cartography to discuss the ticket
- [ ] Obtain a staging link artifact - mentioned in request
- [ ] a11y Support: Staging evaluation of these alerts in-place on the My HealtheVet landing page
- [ ] a11y Support: an assessment of whether there would be any issues with placement / focus management order of these alerts on the main entry point pages of affected tool applications (Medications, medical records, and Secure Messages).
- [ ] Document Findings
- [ ] Review with MHV Cartography Team
```
```[tasklist]
### Acceptance Criteria
- [ ] An evaluation has been completed on the alerts in-place on the MHV landing page.
- [ ] An assessment has been completed noting any issues with placement / focus management order of the alerts on the main entry point pages of affected tool applications (Medications, medical records, and Secure Messages).
- [ ] All outstanding questions addressed with the team.
```
```[tasklist]
## Ticket Updates (CAIA Internal)
- [x] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support)_
- [x] _Label date in the `Open Date` field_
- [x] _Label with `Estimate` (level of effort expected for this ticket)_
- [x] _Add `Assignee(s)` name(s) to ticket_ (include Naomi)
- [x] _Remove assignee (Martha) post priority assignment - unless otherwise indicated_
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [x] _Select a `Priority Level`_ - Martha (high)
- [x] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
```

```[tasklist]
### Related Tickets
- [ ] #99016
```

**Comment by rileyorr** [comment]
Update on this work: I'm waiting on the Cartography team to let me know when the alerts are available to be tested in staging.

**Comment by rileyorr** [comment]
I completed this work a while ago so I'm going to close the ticket - I think it may have gotten lost in the shuffle when we transferred boards.
